server/src/modules/persisters/prompts-persisters.ts

```
@@ -0,0 +1,130 @@
+import { Context } from '../../config/context';
+import { BaseModel } from '../../database/base-model';
+import DatabasePrompts, { DatabasePromptsInitializer, DatabasePromptsId } from '../../types/database/DatabasePrompts';
+
+export interface PromptsModel extends DatabasePrompts {}
+
+export class PromptsModel extends BaseModel {
```

Biome (1.9.4)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)